layers/xdebug/Dockerfile

```diff
@@ -1,18 +1,18 @@
 ARG PHP_VERSION
-FROM bref/build-php-$PHP_VERSION:1.7.4 AS ext
+FROM bref/build-php-$PHP_VERSION:1.7.14 AS ext
 
 RUN set -xe; \
   if [ $PHP_VERSION != "82" ] ; \
   then \
-    pecl install xdebug; \
+  pecl install xdebug; \
   else \
-    cd /tmp; \
-    git clone --depth=1 https://github.com/xdebug/xdebug.git; \
-    cd xdebug; \
-    phpize; \
-    ./configure; \
-    make; \
-    make install; \
+  cd /tmp; \
+  git clone --depth=1 https://github.com/xdebug/xdebug.git; \
+  cd xdebug; \
+  phpize; \
+  ./configure; \
+  make; \
+  make install; \
```

Comment:
Formatting lost here, that might make it harder to maintain. Also xdebug compatible with 8.2 was released, so the if could actually be removed?

Reply:
@mnapoli good catch!
I can also use in place the new heredoc docker feature :wink: 
Let me a moment I will push that.

Reply:
@mnapoli done in https://github.com/brefphp/extra-php-extensions/pull/376/commits/0541f3336563be3df6d78a815cf9bb096ba1ca7a

Reply:
Oh nice feature, I love it!

But also there's a stable xdebug release for PHP 8.2, so maybe we don't want to compile from master anymore?

Reply:
Yup, let's :fire: that

Reply:
done in https://github.com/brefphp/extra-php-extensions/pull/376/commits/97c834bd2b321ca1c6a46c72520ff57c8118a20c

Reply:
Looks Good To Merge I think, can't wait to use php 8.2 on our app with Bref!